src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,20 @@
+<footer class="footer">
+    <div class="container">
+      <div class="footer-content">
+        <div class="footer-logo">
+          <!-- You can replace the img src with the path to your actual logo -->
+          <img src="https://logogen.gdscasu.com/logos/gdsc-logo.png" alt="Logo" />
+        </div>
+        <div class="footer-links">
+          <a href="link-to-events" class="footer-link">Events</a>
+          <a href="link-to-discord" class="footer-link">Discord</a>
+          <a href="link-to-bevy" class="footer-link">Bevy</a>
```

Comment:
Remove class="link-to-bevy", class="link-to-discord", class="link-to-events". These classes aren't created in your css file so they are not doing anything.

Further, update the href's to the actual links. For events, it should be "/events". For the other two, it should be the full link to the bevy and the full link to the discord.